india_stocks_api/internal/angel/api/data.py

```
@@ -331,7 +331,7 @@ def _process_quotes_batch(self, symbols: list) -> list:
         return skipped_symbols + results
 
     def get_history(self, symbol: str, exchange: str, interval: str,
-                   start_date: str, end_date: str) -> pd.DataFrame:
+                   start_date: str, end_date: str, include_oi: bool = None) -> pd.DataFrame:
```

Ruff (0.15.1)

[warning] Unused method argument: `include_oi`

(ARG002)